content/en/mobile/_index.md

```diff
@@ -80,6 +78,62 @@ On the Incidents page, you can view, search and filter all incidents that you ha
 3. Give your incident a title, severity, and commander.
 
 
+## Widgets
+
+### Monitor Saved Views
+
+You can view your Monitor [Saved Views][18] from your home screen with Datadog widgets. Tap on any Saved View cell to open the “Monitor Search” screen in the app, with your Saved View already filled in.
+
+**Note**: If you do not have any Monitor Saved Views, the widget will default to show you all Monitors. 
+
+#### Create a widget
+**iOS**:
```

Comment:
These would work better as tabs:

```
{{< tabs >}}
{{% tab "iOS" %}}

Content

{{% /tab %}}
{{% tab "Android" %}}

Content

{{% /tab %}}
{{< /tabs >}}
```

Reply:
I think the commit I made fixes this, but I am not sure